homeassistant/components/roon/media_player.py

```diff
@@ -148,8 +138,10 @@ def group_members(self):
         return [self._server.entity_id(roon_name) for roon_name in roon_names]
 
     @property
-    def device_info(self) -> DeviceInfo:
+    def device_info(self) -> DeviceInfo | None:
```

Comment:
Why the change in logic here?

Reply:
Because `unique_id` is being initialised as `None`, which is not allowed in the identifiers.
So until the `unique_id` is available, it shouldn't return a `DeviceInfo`.